backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/deliveryplanning/DeliveryPlanningCreateRequest.java

```diff
@@ -69,13 +69,18 @@
 
 	boolean isB2B;
 
-	@NonNull DeliveryPlanningType deliveryPlanningType;
+	boolean isProcessed;
+
+	boolean isClosed;
```

Comment:
are u creating a delivery planning directly Closed or Processed?

Reply:
For now, no. Taking them out.